R/advanced_search.R

```diff
@@ -0,0 +1,100 @@
+#' Advanced search of CRAN packages
+#'
```

Comment:
this is an empty shell for now, but was this the interface you had in mind @gaborcsardi?